### PR TITLE
feat: Create Onsite Order for Tickets

### LINKED ISSUE #1012: Implement Orders Endpoint
### Feature Request

The following features should be implemented regarding Orders endpoint, the features are consistent with eventbrite.

- [x]  List Orders Under Event

- [x]  Order Details

- [ ]  Create Order

**Child Issue**
#522 
   
**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Masquerade0097**
Fixes #1012 

Checklist:

- [ ] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: 
1. Select from multiple tickets available for event.
2. Create Onsite order with onsite attendees.
3. Option to clear the selected tickets.

Screenshots for the change:
ezgif-1-596a976fc6

### CODE REVIEW

**Comment by Masquerade0097** [comment]
@iamareebjamal @sridharjajoo @mishuvs Please review.

**Comment by iamareebjamal** [comment]
Show currency. Also, there is no visual feedback as to when user is clicking somewhere

**Comment by Masquerade0097** [comment]
@iamareebjamal I've made the changes. Here is the new gif.
zzzzz

**Comment by sridharjajoo** [comment]
@Masquerade0097 I think the background can be made blue to make it more consistent with app design

**Comment by Masquerade0097** [comment]
@sridharjajoo I've made the changes.

File: app/src/main/java/com/eventyay/organizer/core/orders/create/CreateOrderViewModel.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+public class CreateOrderViewModel extends ViewModel {
+
+    private final OrderRepository orderRepository;
+    private final TicketRepository ticketRepository;
+    private final CompositeDisposable compositeDisposable = new CompositeDisposable();
+
+    private final MutableLiveData<Float> orderAmount = new MutableLiveData<>();
+    private final MutableLiveData<Boolean> progress = new MutableLiveData<>();
+    private final MutableLiveData<String> success = new MutableLiveData<>();
+    private final MutableLiveData<String> error = new MutableLiveData<>();
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/orders/create/CreateOrderViewModel.java]: `SingleEventLiveData`

File: app/src/main/java/com/eventyay/organizer/core/orders/create/CreateOrderViewModel.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+public class CreateOrderViewModel extends ViewModel {
+
+    private final OrderRepository orderRepository;
+    private final TicketRepository ticketRepository;
+    private final CompositeDisposable compositeDisposable = new CompositeDisposable();
+
+    private final MutableLiveData<Float> orderAmount = new MutableLiveData<>();
+    private final MutableLiveData<Boolean> progress = new MutableLiveData<>();
+    private final MutableLiveData<String> success = new MutableLiveData<>();
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/orders/create/CreateOrderViewModel.java]: `SingleEventLiveData`

**Pull request merged**